homeassistant/components/light/isy994.py

```diff
@@ -72,7 +72,9 @@ def turn_on(self, brightness=None, **kwargs) -> None:
     @property
     def state_attributes(self):
```

Comment:
ISYDevice shouldn't override state attributes either. It is allowed to overwrite device state attributes. Change that and don't change this light platform. It looks as the properties here already return correctly so the attributes will be collected in the state attributes in the light ABC class. Then in the entity class state attributes and device state attributes will be merged.